stream/storage/impl/src/main/java/org/apache/bookkeeper/stream/storage/impl/metadata/RootRangeStoreImpl.java

```diff
@@ -64,6 +66,8 @@
 public class RootRangeStoreImpl
     implements RootRangeStore {
 
+    private static final Logger logger = LoggerFactory.getLogger(RootRangeStoreImpl.class);
```

Comment:
You do not have to create this. 
I think you can log just like that. Creation of logger is taken care of by @Slf4j
log.info("My log {}", myVar)